NBXplorer/Repository.cs

```diff
@@ -809,6 +856,26 @@ private static SavedTransaction ToSavedTransaction(Network network, ReadOnlyMemo
 			t.Transaction.PrecomputeHash(true, false);
 			return t;
 		}
+		public async Task<MultiValueDictionary<OutPoint, KeyPathInformation>> GetKeyInformations(OutPoint[] outPoints)
+		{
+			MultiValueDictionary<OutPoint, KeyPathInformation> result = new MultiValueDictionary<OutPoint, KeyPathInformation>();
+			if (outPoints.Length == 0)
+				return result;
+			var outPointToScript = await GetOutPointToScript(outPoints);
```

Comment:
Since `GetOutPointToScript` is only used here, I would advise to get rid of it completely and iterates through the rows directly to this methods, adding to the `MultiValueDictionary` also the way.

Reply:
Ok, I can remove the function. But it could be required in the future for other uses, right?